Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -352,8 +353,9 @@ final class EpoxyHostingContent<RootView: View>: ObservableObject {
 /// The object that is used to communicate values to SwiftUI views within an
 /// `EpoxySwiftUIHostingController`, e.g. layout margins.
 final class EpoxyHostingEnvironment: ObservableObject {
-  @Published var layoutMargins = EdgeInsets()
+  @Published var hostingController: UIViewController? = nil
```

Comment:
Does this need to be published? i.e. do we ever change it to a new value later on?

Reply:
I believe we want this to be published because we want to reload the wrapped View once the hosting controller is set on line 74.

Reply:
Hmm is it even possible for the view to be rendered before the hosting controller is initialized though? Since the hosting controller is responsible for rendering the view, it seems like that would never happen.

Reply:
Ah, I see. You're right.